tests/conftest.py

```diff
@@ -377,3 +380,14 @@ async def iframe_id(websocket, context_id: str, html_iframe_same_origin, html):
     await goto_url(websocket, iframe_id, html("<h1>FRAME</h1>"))
 
     return iframe_id
+
+
+@pytest_asyncio.fixture
+async def user_context(websocket):
+    """Create a new user context and return its id."""
+    result = await execute_command(websocket, {
+        "method": "browser.createUserContext",
+        "params": {}
+    })
+
+    return result['userContext']
```

Comment:
shall we add auto-remove?

Reply:
We don't clean up the browser state between tests, so no need in this yet.